examples/components/CCFRAUD/preprocessing/conda.yaml

```diff
@@ -1,16 +0,0 @@
-name: ccfraud_preprocess_conda_env
-channels:
-  - defaults
-  - pytorch
-dependencies:
-  - python=3.8
-  - pip=22.2.2
-  - pytorch=1.12.1
-  - torchvision=0.13.1
-  - cudatoolkit=11.3
-  - pip:
-    - azureml-mlflow
```

Comment:
I assume this file should not have been deleted, right?

Reply:
actually I deleted it intentionally, in the training scripts, there are very minimal dependencies that are all present in one of the AzureML curated environment. Using a curated environment shortcuts the build step and skips 10 mins of run time